src/UI/resources/css/components/tables.css

```diff
@@ -132,14 +132,7 @@ table {
 /* Sticky Columns */
 th,
 td {
-  &.sticky-col--right,
-  &.sticky-col--left {
+  &.sticky-col {
     @apply sticky backdrop-blur;
```

Comment:
```suggestion
    @apply lg:sticky backdrop-blur;
```
предлагаю убрать стики колонок на мобильных устройствах

Reply:
А есть вообще смысл тогда?
Как-будто неправильно не делать на мобилках стики. Или отдельный метод под них типо ->sticky(xs: true)

Reply:
Согласен, что в мобильном тоже нужны стики. Но на данном этапе это усложнит данную задачу, основная проблема состоит в кнопках, они на мобильных устройствах занимают почти половину экрана (их нужно сворачивать в выпадающее меню наверное). Поэтому и предложил просто исключить.

Reply:
Да, но наверное все-таки md, а не lg. Потестирую

Reply:
по проекту в основном идет lg